src/components/books/books.html

```diff
@@ -0,0 +1,12 @@
+<div class="main-container">
+    <div class="search">
+      <input type="text" ng-model="searchText" class="search-area" placeholder="Buscá un libro por titulo o autor ">
+    </div>
+    <div class="detail">
+        <div class="book-container" ng-repeat="book in $ctrl.books | searchBook: searchText">
+          <img class="cover"  ng-src='{{book.image_url ? book.image_url : $ctrl.bookCover}}' alt="Portada" />
```

Comment:
You can change this to:
```suggestion
          <img class="cover" ng-src="{{ book.image_url || $ctrl.bookCover }}" alt="Portada" />
```
That way, you don't need the ternary